### PR TITLE
clean up jshint errors

### PR DESCRIPTION
**Pull request by slajax**
## overview
 
This PR simply cleans up the jshint introduced in `2.0` as defined in #43 without removing placeholder code so that the test coverage will actually be what passes or fails the tests in travis. 

## tasks
 - [x] suppress or clean up all jshint errors

## test output

```
> feathers-cli@2.0.0-beta jshint /Users/kylec/repos/feathers-cli
> jshint src/. test/. --config


> feathers-cli@2.0.0-beta mocha /Users/kylec/repos/feathers-cli
> mocha test/ --compilers js:babel-core/register



  feathers-cli
    generator-feathers registration
      1) has registered the generator name
      2) registers all namespaces

  feathers-cli
    ✓ is CommonJS compatible (271ms)
    ✓ basic functionality


  2 passing (307ms)
  2 failing

  1) feathers-cli generator-feathers registration has registered the generator name:
     TypeError: Cannot read property 'getGeneratorNames' of undefined
      at Context.<anonymous> (test/commands/generate.test.js:13:25)

  2) feathers-cli generator-feathers registration registers all namespaces:
     TypeError: Cannot read property 'namespaces' of undefined
      at Context.<anonymous> (test/commands/generate.test.js:26:43)
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,8 @@
     "metalsmith": "^2.1.0",
     "ora": "^0.2.3",
     "request": "^2.72.0",
-    "root-check": "^1.0.0"
+    "root-check": "^1.0.0",
+    "vorpal": "^1.11.4"
```
Comment (ekryski) [inline package.json]: any reason why vorpal is back in here?

Comment (ekryski) [inline package.json]: Ah I know why. The tests are fucked. They are remnants of the old CLI. I'll just merge this and tidy things up.

Comment (slajax) [inline package.json]: Yeah, I added it back in so the tests would just execute in whatever format they were left in, but it seemed like they were not relevant anymore. Let me know if I can help with writing the new test suite.

Comment (ekryski) [inline package.json]: I added some tests yesterday and started working on hooks/middleware. I'll push up or PR them in soon (thinking either today or tomorrow).

**Pull request merged**